graphene_django/tests/test_utils.py

```diff
@@ -51,7 +51,9 @@ def runTest(self):
             pass
 
     tc = TestClass()
+    tc._pre_setup()
     tc.setUpClass()
```

Comment:
I'm not sure why `.setUpClass` had to be called manually here but for same reason, `._pre_setup()` should be called to. Otherwise, Django's `TestCase` doesn't populate `TestCase.client`.